### PR TITLE
Adding Fedora (33) support, fixing sudo make install

### LINKED ISSUE #182: libplist library name can be either `plist` or `plist-2.0`
I had to modify the file `lib/CMakeLists.txt` as follows to make the compilation work:

```diff
diff --git a/lib/CMakeLists.txt b/lib/CMakeLists.txt
index e7071a5..233c752 100755
--- a/lib/CMakeLists.txt
+++ b/lib/CMakeLists.txt
@@ -9,7 +9,7 @@ add_library( airplay
         ${DIR_SRCS}
         )
 
-find_library( LIBPLIST plist )
+find_library( LIBPLIST plist-2.0 )
 
 target_link_libraries( airplay
            pthread
```

I'm using Fedora 33 with `libplist-2.2.0`. It would be helpful to support both library names.

### PR DESCRIPTION
**Pull request by ninjaz5736**
- Fixes #182 - mentions plist-2.0 in lib/CMakeLists.txt
- Update README to show Fedora 33 Instructions
- Change install target in (root/)CMakeLists.txt to point to build directory, enabling successful ```sudo make install```
- Update README to include mention of ```sudo make install```

I haven't been able to test anything on other distributions so I can't say I haven't broken anything, but it *shouldn't* have!

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -19,4 +19,4 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${RENDERER_FLAGS}" )
 add_executable( rpiplay rpiplay.cpp)
 target_link_libraries ( rpiplay renderers airplay )
 
-install(PROGRAMS rpiplay DESTINATION bin)
+install(PROGRAMS build/rpiplay DESTINATION bin)
```
Comment (pallas) [inline CMakeLists.txt:22]: Are you sure this is right? Shouldn't `install` already be relative to the build path.

**Comment by pallas** [comment]
The first two look great, thanks! The last one breaks my install command.

File: CMakeLists.txt (hunk 1 above)
Comment (ninjaz5736) [reply CMakeLists.txt:22]: Strange... my install command seems to be relative to the base directory. The generated build/cmake_install.cmake line 46 references the (non-existent) root/rpiplay executable for me, hence my "No such file or directory" woes.
I'm not a cmake pro so I'm not going to be much help here. I'm happy to test suggestions though! Shall I redo the pull without this edit / the respective README edit, or is there an easy way to edit this pull?

My generated build/cmake_install.cmake file for reference:
```
# Install script for directory: /home/builduser/RPiPlay

# Set the install prefix
if(NOT DEFINED CMAKE_INSTALL_PREFIX)
  set(CMAKE_INSTALL_PREFIX "/usr/local")
endif()
string(REGEX REPLACE "/$" "" CMAKE_INSTALL_PREFIX "${CMAKE_INSTALL_PREFIX}")

# Set the install configuration name.
if(NOT DEFINED CMAKE_INSTALL_CONFIG_NAME)
  if(BUILD_TYPE)
    string(REGEX REPLACE "^[^A-Za-z0-9_]+" ""
           CMAKE_INSTALL_CONFIG_NAME "${BUILD_TYPE}")
  else()
    set(CMAKE_INSTALL_CONFIG_NAME "")
  endif()
  message(STATUS "Install configuration: \"${CMAKE_INSTALL_CONFIG_NAME}\"")
endif()

# Set the component getting installed.
if(NOT CMAKE_INSTALL_COMPONENT)
  if(COMPONENT)
    message(STATUS "Install component: \"${COMPONENT}\"")
    set(CMAKE_INSTALL_COMPONENT "${COMPONENT}")
  else()
    set(CMAKE_INSTALL_COMPONENT)
  endif()
endif()

# Install shared libraries without execute permission?
if(NOT DEFINED CMAKE_INSTALL_SO_NO_EXE)
  set(CMAKE_INSTALL_SO_NO_EXE "0")
endif()

# Is this installation the result of a crosscompile?
if(NOT DEFINED CMAKE_CROSSCOMPILING)
  set(CMAKE_CROSSCOMPILING "FALSE")
endif()

# Set default install directory permissions.
if(NOT DEFINED CMAKE_OBJDUMP)
  set(CMAKE_OBJDUMP "/usr/bin/objdump")
endif()

if("x${CMAKE_INSTALL_COMPONENT}x" STREQUAL "xUnspecifiedx" OR NOT CMAKE_INSTALL_COMPONENT)
  file(INSTALL DESTINATION "${CMAKE_INSTALL_PREFIX}/bin" TYPE PROGRAM FILES "/home/builduser/RPiPlay/rpiplay")
endif()

if(NOT CMAKE_INSTALL_LOCAL_ONLY)
  # Include the install script for each subdirectory.
  include("/home/builduser/RPiPlay/build/lib/curve25519/cmake_install.cmake")
  include("/home/builduser/RPiPlay/build/lib/ed25519/cmake_install.cmake")
  include("/home/builduser/RPiPlay/build/lib/playfair/cmake_install.cmake")
  include("/home/builduser/RPiPlay/build/lib/llhttp/cmake_install.cmake")
  include("/home/builduser/RPiPlay/build/lib/cmake_install.cmake")
  include("/home/builduser/RPiPlay/build/renderers/cmake_install.cmake")

endif()

if(CMAKE_INSTALL_COMPONENT)
  set(CMAKE_INSTALL_MANIFEST "install_manifest_${CMAKE_INSTALL_COMPONENT}.txt")
else()
  set(CMAKE_INSTALL_MANIFEST "install_manifest.txt")
endif()

string(REPLACE ";" "\n" CMAKE_INSTALL_MANIFEST_CONTENT
       "${CMAKE_INSTALL_MANIFEST_FILES}")
file(WRITE "/home/builduser/RPiPlay/build/${CMAKE_INSTALL_MANIFEST}"
     "${CMAKE_INSTALL_MANIFEST_CONTENT}")
```

**Comment by pallas** [comment]
I already merged the other two. For the install thing, I typically build in-tree (yeah, yeah, I know) but let me try an out of tree build and see what changes. Do you have additional CMAKE stuff in your env? What version of cmake are you using?

**Comment by ninjaz5736** [comment]
Ahh that would make sense, the instructions say have it done inside the build directory so that's probably why there's the mismatch. I'm using cmake version 3.18.4 on Fedora 33 Workstation (64-bit), no variables in my env AFAIK, I only installed cmake a few days ago for another unrelated project! I've just tried an in-tree build and it works flawlessly, so probably just needs either documentation or that file updating :-)

**Comment by pallas** [comment]
Ok, I see the problem. Fixed upstream.

**Pull request closed without merging**